docs/configuration.rst

```diff
@@ -25,136 +25,171 @@ Mapcrafter render something (let's call it ``render.conf``):
-interface of the output HTML-File. This can be anything suitable to identify
-this map. In contrast to that the world and map names in the sections are used
-for internal representation and therefore should be unique and contain only
-alphanumeric chars and underscores.
-
-When you have now your configuration file you can render your worlds with (see
-:ref:`command_line_options` for more options and usage)::
-
-    mapcrafter -c render.conf
+You can find other `example maps <https://github.com/mapcrafter/mapcrafter/wiki/Example-maps>`_ 
+on the Github Wiki. Other configuration examples may be available there.
 
 There are tons of other options to customize your rendered maps. Before a
-reference of all available options, here is a quick overview of interesting
-things you can do:
+reference of fundamentals and all available options, here is a quick overview 
+of interesting things you can do:
 
 * Default view / zoom level / rotation in web interface
 * World cropping (only render specific parts of your world)
 * Block mask (skip rendering / render only specific types blocks)
 * Different render views, render modes, overlays
-* Use custom texture packs, texture sizes, apply a blur effect to textures
+* Use custom texture packs, or texture sizes
 * Custom tile widths
 * Different image formats
 * Custom lighting intensity
 
 
+Configuration File Fundamentals
+===============================
+
+Configuration files consist of several sections (e.g. ``[section]``) and 
+*key+value* pairs (e.g. ``key = value``). Sections are named after a colon
+(eg: ``[section:name]``). Lines starting with a hash (``#``) are ignored and can
+be used as comments or notes.
+
+There are four sections which can configure Mapcrafter output:
+
+* The General section (values entered before any section),
+* World sections (e.g. sections starting with ``world:``),
+* Map sections (e.g. sections starting with ``map:``),
+* and Marker sections (e.g. sections starting with ``marker:``, also see :ref:`markers`).
+
+The *General* section specifies the ``output_dir`` where your map(s) will be
+rendered. A single ``index.html`` will created in this folder, which you can
+open in your web-browser and view all maps in your config. See `General Options`_ 
+for more details.
+
+Every *World* section represents a Minecraft world you want to render and needs 
+an ``input_dir``. You can also choose the dimension to render and a variety of 
+cropping options. See `World Options`_ for more details.
+
+Every *Map* section represents an actual rendered map of a Minecraft world. You
+can specify things like rotation of the world, render view, render mode, spritemaps, 
+texture size, etc. Each ``[map:...]`` will render a unique map you can
+view in your web-browser. Oh, and you need at least one map, or Mapcrafter
+can't render anything! See `Map Options`_ for more details.
+
+Marker sections allow you to add map markers based on Minecraft signs. It is
```

Comment:
`It is are outlined...`. It is are maybe a typo? :speak_no_evil:

Reply:
Well spotted sir!